### PR TITLE
[4.3] Normalize styling of inputs

### PR DESCRIPTION
**Pull request by wilsonge**
### Summary of Changes
Minor rework of input styling across both backend and frontend templates. The aim of this is to use more of the native bootstrap variables so that our rendering is more consistent. This also makes it easier for people to reuse our SCSS files that we distribute by using more of the bootstrap mixins (e.g. using the box-shadow and border-radius mixins).

### Testing Instructions
Go through native select, input, choices (both multiselect and single select - e.g. category and tags form fields in frontend editing) form fields in both cassieopia and atum. The changes should not be noticeable before and after.

For bonus points:
- Only find an extension still using chosen and test that too.
- Set `enable-rounded` variable to false in either casseopia or atum. Before patch only native BS fields were square and the rest were round. After they all act the same way.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/media_source/templates/administrator/atum/scss/_variables.scss [hunk 1]
```diff
@@ -94,8 +91,8 @@ $colors: (
   border:                          $gray-400,
   "white":                         $white, // the key here must be in quotes to avoid scss compilation warnings.
   white-offset:                    $whiteoffset,
-  focus:                           $focuscolor,
-  focus-shadow:                    $focusshadow,
+  focus:                           #39f,
+  focus-shadow:                    #eaeaea,
```
Comment (dgrammatiko) [inline build/media_source/templates/administrator/atum/scss/_variables.scss:95]: Could we reuse some colour from the existing vars here?
Or add these colours to the colour array?

Comment (wilsonge) [reply build/media_source/templates/administrator/atum/scss/_variables.scss:95]: This is the color array :D And it's the same colours as before (see removed lines 16 & 19). I've just ensured all ref's now point to the CSS var

Comment (dgrammatiko) [reply build/media_source/templates/administrator/atum/scss/_variables.scss:95]: Pfff, I should have expanded the code above. All good

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on e74918bd18bd0986b6201e20277c5a0300079790<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39781">issues.joomla.org/tracker/joomla-cms/39781</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on e74918bd18bd0986b6201e20277c5a0300079790<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39781">issues.joomla.org/tracker/joomla-cms/39781</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39781">issues.joomla.org/tracker/joomla-cms/39781</a>.</sub>

**Comment by obuisard** [comment]
Thank you, George @wilsonge for this PR!

**Pull request merged**